File: python3/1343.py
class Solution:
    def numOfSubarrays(self, n: List[int], k: int, t: int) -> int:
        p = 0
        c = 0
        t *= k
        for i in range(k):
            p += n[i]
        if p >= t:
            c += 1
        for i in range(k, len(n)):
            p = p - n[i - k] + n[i]
            if p >= t:
                c += 1
        return c

File: c/1343.c
int numOfSubarrays(int* n, int l, int k, int t) {
    int p = 0, c = 0;
    t *= k;
    for (int i = 0; i < k; i++)
        p += n[i];
    if (p >= t)
        c++;
    for (int i = k; i < l; i++) {
        p = p - n[i - k] + n[i];
        if (p >= t)
            c++;
    }
    return c;
}
